Bottom: 73dca28dd2d7c6108756683d95a00aa0c7df75e7
Top:    e35f13e7defae8cc0c8c759a5655661cce731701
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 02:32:44 +0000

Refresh of work-in-progress-on-subcomposi

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 45f753b..e872e87 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -565,8 +565,7 @@ ghid_set_layer (const char *name, int group, int empty)
 	    return TEST_FLAG (SHOWMASKFLAG, PCB);
 	  return 0;
 	case SL_SILK:
-          gport->trans_lines = TRUE;
-//          gport->trans_lines = FALSE;
+	  gport->trans_lines = TRUE;
 	  if (SL_MYSIDE (idx) /*|| pinout */ )
 	    return PCB->ElementOn;
 	  return 0;
